### DISCUSSION TITLE
Clicable URLs

### DISCUSSION THREAD

**Discussion by TimothyMRead** [coddingtonbear/obsidian-web#94] [General]
The URLs that show where the webpage has been copied are just text. Is there a way to make them actual links back to the original page?

**Comment by coddingtonbear** [comment]
I’m not sure I follow; what URL are you talking about?

**Reply by TimothyMRead** [reply]
Sorry for my lack of clarity. At the top of each page there is a summary section, such as this one:

```
---
page-title: "Inflammation May Be the Culprit Behind Deadly Diseases | Time"
url: https://time.com/6269070/inflammation-deadly-diseases/
date: "2023-04-11 16:57:57"
---
```

However, the url there is just plain text. To be able to go back to the original page I have to copy it and paste it into a browser. It would be nice if it were a link to the original page...

**Reply by TimothyMRead** [reply]
Aghhh. Here, in this forum, the url is converted to a link, but in the page in Obsidian it is just plain text...

**Reply by coddingtonbear** [reply]
For future reference, you can surround your code samples in three backticks before and after your code sample to make it not be interpreted as markdown.  I've updated your comment above to reflect that.

So: it looks like that particular URL you're seeing is in the "frontmatter" section of your document.  I don't know exactly what rules Obsidian uses for determining how to render frontmatter, and it's probably dependent on what plugins you're using, but given that frontmatter fields are intended to be used as metadata instead of actual, interactive parts of your document, figuring out a way of making that URL clickable is not what I'd recommend.

Instead: what I'd recommend doing is updating your template such that your document URL appears again in the actual body of your document (i.e. any line below the second `---`).

**Reply by TimothyMRead** [reply]
Wow!!! Many thanks. I have just learned so much... Modifying the position of the url in the Capture page snapshot has done the trick.
